Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -197,6 +193,14 @@ The password must meet at least three out of the following four complexity rules
 3. at least one digit (0-9)
 4. at least one special character (punctuation)
 
+**Alternative**
```

Comment:
It comes from your discussion with @sushi2k (Sjord : "The dictionary check is an _alternative_ for complexity rules."), I just based my proposal on this, nothing new. Let me know what's best.